templates/registration/login.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>Login</h1>
+    {% if error %}
```

Comment:
The `error` variable is used to display error messages, but it is not a standard part of Django's authentication context. Ensure that this variable is being set in the view that renders this template if you intend to use it for error messages.